meet-data/rpu/1701/meet.csv

```diff
@@ -0,0 +1,2 @@
+Federation,Date,MeetCountry,MeetState,MeetTown,MeetName
+IPL,2017-04-20,Russia,,Dolgoprudny,Чемпионат Европы
```

Comment:
The MeetName should be translated to idiomatic English. This particular meet is named "European Championship".

Also, is RPU affiliated with IPL? I don't know anything about RPU.